src/d3a/models/myco_matcher/__init__.py

```diff
@@ -0,0 +1,31 @@
+from d3a.constants import DOUBLE_SIDED_ALGO
+from d3a.models.myco_matcher.pay_as_bid import PayAsBidMatch
+from d3a.models.myco_matcher.pay_as_clear import PayAsClear
+
+
+class MycoMatcher:
+    def __init__(self):
+        self.bid_offer_pairs = []
+        if DOUBLE_SIDED_ALGO == 1:
+            self.match_algo = PayAsBidMatch()
+        elif DOUBLE_SIDED_ALGO == 2:
+            self.match_algo = PayAsClear()
+
+    @staticmethod
+    def sorting(obj, reverse_order=False):
+        if reverse_order:
+            # Sorted bids in descending order
+            return list(reversed(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate)))
```

Comment:
```suggestion
            return list(sorted(
                obj.values(),
                key=lambda b: b.energy_rate, reverse=True))
```

Reply:
- What kind of object is `obj`? I would add this information as type hint.
- If `obj` is a `dict`, you can directly use `sorted()` without casting to `list`.
- `sorted()` has a nice `reverse` parameter that you can use to replace `reversed`.